consul/consul.nuspec

```diff
@@ -12,7 +12,12 @@
     <packageSourceUrl>https://github.com/cleung2010/chocolatey-packages/tree/master/consul</packageSourceUrl>
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
     <description>Consul is a tool for service discovery and configuration. Consul is distributed, highly available, and extremely scalable.
-Uses NSSM as the service wrapper. More info at: https://nssm.cc/</description>
+Uses NSSM as the service wrapper. More info at: https://nssm.cc/
+
+#### Package Parameters
```

Comment:
https://github.com/chocolatey/choco/wiki/How-To-Parse-PackageParameters-Argument#add-package-parameter-information-to-the-description